Description: Add content_rating field to appdata file
Author: Benjamin Barenblat <bbaren@debian.org>
Forwarded: no
--- a/data/io.github.TransmissionRemoteGtk.appdata.xml.in
+++ b/data/io.github.TransmissionRemoteGtk.appdata.xml.in
@@ -56,4 +56,5 @@
     <id>uk.org.eth0.transmission-remote-gtk.desktop</id>
   </provides>
   <update_contact>tingping_at_fedoraproject.org</update_contact>
+  <content_rating type="oars-1.0" />
 </component>
